string order_id
string order_time
string item
string quantity
string cost
string city
string lon
string lat
